task4/task4.ipynb

```diff
@@ -22,28 +22,45 @@
     "# 4a"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- $p_{c_k}$ : Active probability from constant retention rate. $p_{c_1} = 1$\n",
+    "- $r_c$ : Constant Retention rate"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\\mathbb{E}_c [user \\space months] = \\sum_{k=1}^{\\inf}k \\cdot p_{c_{k}} = \\sum_{k=1}^{\\inf}k \\cdot r_c^{k-1}$$"
```

Comment:
Notice the {k-1} in the exponent of the constant retention rate when calculating the expected value. Essentially include 1*1 in the expected value since 100% of the acquired customers survive the first month. Results in higher Expected value for constant rate than before